documentation/asciidoc/computers/raspberry-pi/boot-nvme.adoc

```diff
@@ -19,42 +17,13 @@ If you need to connect the NVMe drive to a PC or Mac you can use a USB adaptor:
 
 === Required Software
 
-To boot from NVMe you need pre-release versions of the bootloader, VideoCore firmware and Raspberry Pi OS Linux kernel.
+To boot from NVMe you need a recent version of the bootloader (July 2021), and pre-release versions of the VideoCore firmware and Raspberry Pi OS Linux kernel.
```

Comment:
> pre-release versions of the VideoCore firmware

How long until this lands in APT? Perhaps it's worth delaying this PR until it no longer requires pre-release firmware? :shrug: